client/package.json

```diff
@@ -50,6 +50,7 @@
 		"@wordpress/format-library": "^1.15.0",
 		"@wordpress/i18n": "^3.10.0",
 		"@wordpress/is-shallow-equal": "^2.0.0",
+		"@wordpress/icons": "^1.2.0",
```

Comment:
2.0.0 has been released, and is already being used (transitively) in Calypso AFAICS!

Reply:
Right you are, thanks. Fixed it up 👍